packages/gatsby-theme-garden/README.md

```diff
@@ -56,3 +56,17 @@ module.exports = {
 | `roamEmail`              |               | Email used to sign into Roam Research                                            |
 | `roamPassword`           |               | Password used to sign into Roam Research                                         |
 | `mdxOtherwiseConfigured` | `false`       | Set this flag `true` if `gatsby-plugin-mdx` is already configured for your site. |
+
+### How to add syntax highlighting support?
```

Comment:
I think we could spin it in a more generic way: "How to inject Mdx components?" and use syntax highlighting as a example maybe?